users/admin.py

```diff
@@ -10,7 +10,7 @@ class CustomUserAdmin(UserAdmin):
     fieldsets = (
```

Comment:
Instead of naming this commit `Remove wrong coding style` write `Fix style in code` :)